Sources/SwiftDocC/Infrastructure/Symbol Graph/SymbolGraphLoader.swift

```diff
@@ -404,9 +425,8 @@ extension SymbolGraph.Symbol.Availability.AvailabilityItem {
     /// - Note: If the `defaultAvailability` argument doesn't have a valid
     /// platform version that can be parsed as a `SemanticVersion`, returns `nil`.
     init?(_ defaultAvailability: DefaultAvailability.ModuleAvailability) {
-        guard let introducedVersion = defaultAvailability.introducedVersion, let platformVersion = SymbolGraph.SemanticVersion(string: introducedVersion) else {
-            return nil
-        }
+        let introducedVersion = defaultAvailability.introducedVersion
+        let platformVersion = (introducedVersion != nil) ? SymbolGraph.SemanticVersion(string: introducedVersion!) : nil
```

Comment:
minor: This pattern (`value != nil ? makeSomething(value!) : nil`) is a good use of [`Optional.map(_:)`](https://developer.apple.com/documentation/swift/optional/map(_:))

```suggestion
        let platformVersion = introducedVersion.map { SymbolGraph.SemanticVersion(string: $0) }
```

Reply:
This would return a double optional. But something I can do is
```
let platformVersion = introducedVersion.map { SymbolGraph.SemanticVersion(string: $0) } ?? nil
```

Reply:
Resolved in https://github.com/swiftlang/swift-docc/pull/1024/commits/6055aa94f29b2e827aafa8598f531adbeccb25e7